### PR TITLE
hostcheck@schorschii: Add manual refresh time settings, add czech translation and .pot format revision

### PR DESCRIPTION
**Pull request by Gujenczek**
Easy manual set refresh time value in menu/setup and visible this value at desklet.

Language update.

I make czech translation in 'po/cs.po', but I don't know how '/home/<user>/.local/share/locale/cs/LC_MESSAGES' working for add this.

@schorschii

### CODE REVIEW

File: hostcheck@schorschii/files/hostcheck@schorschii/desklet.js [hunk 1]
```diff
@@ -160,7 +161,7 @@ MyDesklet.prototype = {
 		let label_size_x = this.defaultWidth * this.scale_size * global.ui_scale;
 		let label_size_y = this.defaultHeight * this.scale_size * global.ui_scale
 		// modify label
-		let statusString = this.statusTagString + "\n" + this.host;
+		let statusString = this.statusTagString + " / " + this.interval + "s\n" + this.host;
```
Comment (schorschii) [inline hostcheck@schorschii/files/hostcheck@schorschii/desklet.js]: @Gujenczek Thanks for these improvements. I only have one thing to say, can you please make the "s" translatable and put the interval in brackets (no slash)?

**Comment by schorschii** [comment]
> I make czech translation in 'po/cs.po', but I don't know how '/home//.local/share/locale/cs/LC_MESSAGES' working for add this.

Please have a look at the [README.md](https://github.com/linuxmint/cinnamon-spices-desklets#translations). You can use `./cinnamon-spices-makepot UUID --install` to test your translations.

**Comment by Gujenczek** [comment]
Brackets is resolded and couple units 'seconds' are translatable.
'cs.po' is good.

**Comment by Gujenczek** [comment]
@schorschii

**Review by schorschii — APPROVED** [review]

**Comment by schorschii** [comment]
Thanks. @brownsr (or any other member of the Mint team) can you please merge this?

**Pull request merged**